srtcore/core.h

```diff
@@ -948,13 +948,15 @@ class CUDT
     int32_t m_iReXmitCount;                      // Re-Transmit Count since last ACK
 
     static const size_t
-                MAX_FREQLOGFA = 2,
+                MAX_FREQLOGFA = 3,
```

Comment:
Put it at the end so that it gets upated when anything is added at the END of the list.